app/managers.py

```diff
@@ -0,0 +1,39 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema_db.sqlite")
+        self.table_name = "actors"
+
+    def all(self) -> list:
```

Comment:
`-> list:`
Maybe better show what in the list you returning ?
For example `-> list[SomeClass]:`